.github/workflows/github-token-rotation-prod.yml

```
@@ -0,0 +1,72 @@
+          # Assume IAM role for GitHub Actions to access AWS resources
+          role-to-assume: ${{ secrets.GITHUB_TOKEN_ROTATION_ROLE_TO_ASSUME_PROD}}
+          aws-region: ${{ secrets.AWS_REGION }}
+
+      - name: Install PyJWT
+        run: |
+          pip3 install PyJWT      
+
+      - name: Rotate GitHub Token
+        env:
+          SECRET_NAME: github-token
+        run: |
```

actionlint

shellcheck reported issue in this script: SC2086:info:24:44: Double quote to prevent globbing and word splitting

(shellcheck)